test/commands/resources/create-issue-input.valid.yaml

```diff
@@ -0,0 +1,59 @@
+sellerName: test
+sellerMarketPlaceAlias: cloufsoft-developer
+accountId: "012345678901"
+productName: myapp
+productCategory: networking
+productUrl: https://test.io/software
+addon:
+  name: myapp
+  versionName: test myapp 0.1.0 mp1
+  version: 1.0.0
+  namespace: test-myapp
+  type: networking
+  helmChartUrl: 709825985650.dkr.ecr.us-east-1.amazonaws.com/test-development/test-myapp-helm:0.1.6
+  containerImagesUrls:
+    - 709825985650.dkr.ecr.us-east-1.amazonaws.com/test-development/test-myapp:0.4.2
```

Comment:
What AWS Account number is this? Lets remove an AWS Account numbers in public repos.